tools/azsdk-cli/Azure.Sdk.Tools.Cli/Tools/Package/VersionUpdateTool.cs

```diff
@@ -0,0 +1,242 @@
+
+            // Validate package path
+            if (string.IsNullOrWhiteSpace(packagePath))
+            {
+                return PackageOperationResponse.CreateFailure("Package path is required and cannot be empty.");
+            }
+
+            if (!Directory.Exists(packagePath))
+            {
+                return PackageOperationResponse.CreateFailure($"Package path does not exist: {packagePath}");
+            }
+
+            // Set default release type to 'beta' if both Version and ReleaseType are empty or null
+            if (string.IsNullOrWhiteSpace(version) && string.IsNullOrWhiteSpace(releaseType))
+            {
+                releaseType = "beta";
+                logger.LogInformation("No version or release type specified. Defaulting to release type: beta");
+            }
+
+            // Discover the repository root
+            var sdkRepoRoot = gitHelper.DiscoverRepoRoot(packagePath);
+            if (sdkRepoRoot == null)
+            {
+                return PackageOperationResponse.CreateFailure("Unable to find git repository root from the provided package path.");
+            }
+
+            logger.LogInformation("Repository root discovered: {SdkRepoRoot}", sdkRepoRoot);
+            var languageService = GetLanguageService(packagePath);
+            if (languageService == null)
+            {
+                return PackageOperationResponse.CreateFailure("Tooling error: unable to determine language service for the specified package path.", nextSteps: ["Create an issue at the https://github.com/Azure/azure-sdk-tools/issues/new", "contact the Azure SDK team for assistance."]);
+            }
+
+            // Check for package type
+            var packageInfo = await languageService.GetPackageInfo(packagePath, ct);
+            if (packageInfo?.SdkType == SdkType.Management)
+            {
+                // For management-plane packages, execute configured version update script
+                var (configContentType, configValue) = await _specGenSdkConfigHelper.GetConfigurationAsync(sdkRepoRoot, SpecGenSdkConfigType.UpdateVersion);
+                if (configContentType != SpecGenSdkConfigContentType.Unknown && !string.IsNullOrEmpty(configValue))
+                {
+                    logger.LogInformation("Found valid configuration for updating version. Executing configured script...");
+
+                    // Prepare script parameters
+                    var scriptParameters = new Dictionary<string, string>
+                    {
+                        { "SdkRepoPath", sdkRepoRoot },
+                        { "PackagePath", packagePath }
+                    };
+
+                    // Add optional parameters if provided
+                    if (!string.IsNullOrWhiteSpace(releaseType))
+                    {
+                        scriptParameters["ReleaseType"] = releaseType;
```

Comment:
Shouldn't we always pass these values even if they are empty?

Reply:
They're optional parameters and don't have to be passed in when they're empty.